bot/seasons/evergreen/snakes/snakes_cog.py

```diff
@@ -1164,6 +1164,7 @@ def predicate(reaction_: Reaction, user_: Member):
     # endregion
 
     # region: Error handlers
+    @Cog.listener()
```

Comment:
Are you sure this counts as a general "event"? I thought it's just the error handler for the commands below.

Reply:
I think you're right.

I thought I checked this but judging from the documentation I believe I did it wrong. Will confirm next time I'm on my laptop.

Reply:
It does nothing, removed